gef-ui/server/controllers/application-details/index.js

```diff
@@ -165,6 +165,7 @@ const stateToPartial = (status, url) => {
     [DEAL_STATUS.UKEF_REFUSED]: 'application-preview',
     [DEAL_STATUS.EXPIRED]: '',
     [DEAL_STATUS.WITHDRAWN]: '',
+    [DEAL_STATUS.CANCELLED]: 'application-preview',
```

Comment:
Do we have a test for this?

Reply:
Added an it.each & this case to `gef-ui\server\controllers\application-details\index.test.js`